### PR TITLE
Adds Lola's Set to the autovend

### PR DESCRIPTION
**Pull request by Kharkov72**
Adds lola's clown set to the autovent, thank you threadman
Untitled
it's under premium because those are some real and squeaking clown shoes
[sprites]
also please god if you can think of better descriptions/names give me them

:cl:
 * rscadd: Adds Lola's Clown Set to the Autovend!

### CODE REVIEW

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -163,6 +163,13 @@
 		qdel(W)
 		qdel(src)
 
+obj/item/clothing/shoes/clown_shoes/lola
+	desc = "Honk!"
```
Comment (DrSnips) [inline code/modules/clothing/shoes/miscellaneous.dm]: >desc before name

This bothers my autism to no end.

Comment (Kharkov72) [reply code/modules/clothing/shoes/miscellaneous.dm]: well the other clown shoes are like that so it's c o n s i s t e n t
(and by that I mean the elf shoes)

**Comment by SonixApache** [comment]
who

**Comment by MadmanMartian** [comment]
who?

**Comment by DrSnips** [comment]
A clown character from ARMS.

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -209,6 +209,14 @@
 	can_flip = 0
 	canstage = 0
 
+obj/item/clothing/mask/gas/lola
```
Comment (ShiftyRail) [inline code/modules/clothing/masks/gasmask.dm]: This is not standard but all other items in the file start with `/`
My autism is midly annoyed by this

**Pull request merged**